__tests__/integration.js

```diff
@@ -0,0 +1,391 @@
+const awsServerlessExpress = require('../index')
+const apiGatewayEvent = require('../example/api-gateway-event.json')
+const app = require('../example/app')
+
+const server = awsServerlessExpress.createServer(app)
+const lambdaFunction = {
+  handler: (event, context) => awsServerlessExpress.proxy(server, event, context)
+}
+
+function clone(json) {
+  return JSON.parse(JSON.stringify(json))
+}
+
+function makeEvent(eventOverrides) {
+  const baseEvent = clone(apiGatewayEvent)
+  const headers = Object.assign({}, baseEvent.headers, eventOverrides.headers)
+  delete eventOverrides.headers
+  const root = Object.assign({}, baseEvent, eventOverrides)
+  root.headers = headers
+  return root
+}
+
+function expectedRootResponse() {
```

Comment:
Helper function for root response since we have multiple tests which hit root route. The same could probably be applied for other routes.